### ISSUE TITLE
'--output yaml' outputs non-YAML Types as Python Objects

### ISSUE THREAD

**Issue by dennisconrad** [cloudreach/sceptre#630]
`--output yaml` outputs non-YAML types as Python objects.  For example:

```yaml
- tst/vpc:
  - !!python/unicode 'OutputKey': SubnetXId
    !!python/unicode 'OutputValue': subnet-X
```

Since the output will never be a Python object, it should be printed as plain YAML.
 
Note: tested with Python 2.7.15

**Comment by dennisconrad** [comment]
PR https://github.com/cloudreach/sceptre/pull/631

**Comment by ngfgrant** [comment]
resolved by #631
